Sources/NIOHTTP2/HTTP2Error.swift

```diff
@@ -467,7 +467,9 @@ public enum NIOHTTP2Errors {
             }
         }
 
-        private final class Storage: Equatable {
+        private final class Storage: Equatable, @unchecked Sendable {
```

Comment:
I mean, this type is very trivially _not_ `Sendable`. Can it be made to be? Same for the other changes here.

Reply:
I'd put the annotation in the wrong place, I've moved it to the broader error type.